platform/ui/src/components/ViewportActionBar/ViewportActionBar.tsx

```diff
@@ -55,79 +84,81 @@ const ViewportActionBar = ({
     return () => clickOutsideListener.remove();
   }, [clickOutsideListener, showPatientInfo]);
 
-  const borderColor = useAltStyling ? '#365A6A' : '#1D205A';
+  /**
+   * Handles what gets hidden and what gets shown during a resize of the viewport.
+   */
+  const resizeCallback = useCallback(() => {
+    if (!componentRootElemRef) {
+      return;
+    }
 
-  let backgroundColor = '#020424';
-  if (useAltStyling) {
-    backgroundColor = '#031923';
-  }
+    const parentElemRefBBox = componentRootElemRef.current.getBoundingClientRect();
+
+    // Show or hide the arrows based on the viewport/root element width.
+    if (parentElemRefBBox.width < arrowsPresentViewportMinWidth) {
+      setShowArrows(false);
+    } else {
+      setShowArrows(true);
+    }
+
+    const studyDateElemBBox = studyDateElemRef.current.getBoundingClientRect();
+    const showPatientInfoElemBBox = showPatientInfoElemRef.current.getBoundingClientRect();
+
+    if (
+      showPatientInfoElemBBox.left - studyDateElemBBox.right <=
+      zeroWidthSeriesDescriptionSpace
+    ) {
+      // The area to display the series description is zero, so don't show the series description element.
+      setShowSeriesDesc(false);
+    } else {
+      setShowSeriesDesc(true);
+    }
+  }, []);
+
+  useResizeObserver(componentRootElemRef?.current, resizeCallback);
 
   return (
     <div
-      className="flex flex-wrap items-center p-2 -mt-2 border-b select-none"
-      style={{
-        borderColor: borderColor,
-        backgroundColor: backgroundColor,
-      }}
+      ref={componentRootElemRef}
+      className="pointer-events-auto select-none text-base flex overflow-visible whitespace-nowrap h-8 items-center px-2 shrink-0"
```

Comment:
`pointer-events-auto` is the key for mouse interactions (i.e. down, up, hover, click, etc.) to work for action bars of inactive viewports. However, as you will see for the SR and SEG status components, sometimes `onClick` does not work despite this.